test/system/app-os/java.spec.ts

```diff
@@ -0,0 +1,20 @@
+import { join as pathJoin } from "path";
```

Comment:
I know it's a bit confusing but this test should probably be under `application-scans` not `app-os` 🙏  `app-os` is about the manifestGlobs.
Actually it's probably a better idea to merge both `app-os` and `application-scans` into only a single directory `application-scans` so it's just less confusing!

Reply:
So I thought that was the purpose of this dir: "app-os" being "app&os". I can change the name if it's confusing.

Reply:
Ah ok, I can see it now!! I haven't noticed that there was already a dir for app scans!!! I'll merge them 👍